include/dgl/runtime/config.h

```diff
@@ -0,0 +1,34 @@
+/*!
+ *  Copyright (c) 2019 by Contributors
+ * \file runtime/config.h
+ * \brief DGL runtime config
+ */
+
+
+#ifndef DGL_RUNTIME_CONFIG_H_
+#define DGL_RUNTIME_CONFIG_H_
+
+namespace dgl {
+namespace runtime {
+
+class Config {
+ public:
+  static Config* Global() {
+    static Config config;
+    return &config;
+  }
+
+  // Enabling or disable use libxsmm for Spmm
+  void enableLibxsmm(bool);
```

Comment:
Make sure you follow the Camel naming style: `EnableLibxsmm`, `IsLibxsmmAvailable` .

Reply:
Fixed